dist/tools/compile_and_test_for_board/compile_and_test_for_board.py

```diff
@@ -33,36 +33,46 @@
 
 ```
 usage: compile_and_test_for_board.py [-h] [--applications APPLICATIONS]
-                                     [--applications-exclude
-                                      APPLICATIONS_EXCLUDE]
+                                     [--applications-exclude APPLICATIONS_EXCLUDE]
                                      [--no-test]
-                                     [--loglevel {debug,info,warning,error,
-                                                  fatal,critical}]
+                                     [--loglevel {debug,info,warning,error,fatal,critical}]
                                      [--incremental] [--clean-after]
+                                     [--compile-targets COMPILE_TARGETS]
+                                     [--test-targets TEST_TARGETS]
                                      [--jobs JOBS]
                                      riot_directory board [result_directory]
 
 positional arguments:
   riot_directory        RIOT directory to test
   board                 Board to test
-  result_directory      Result directory, by default "results"
+  result_directory      Result directory, by default "results" (default:
```

Comment:
the `by default "results"` is redundant with `(default: results)`.

Reply:
I will add a commit to update the docstring to remove the `default`.